rules/style.js

```diff
@@ -10,8 +10,43 @@ module.exports = {
         'import/no-dynamic-require': 'off',
         'import/no-extraneous-dependencies': 'off',
         'import/no-unresolved': 'off',
-        // Enforce indentation of 4 spaces
-        indent: ['error', 4],
+
+        // Enforce indentation of 4 spaces; the third option parameter was copied from Airbnb:
+        // https://github.com/airbnb/javascript/blob/60b96d322277c4c71a21a05caba8eb3320e0e3fa/packages/eslint-config-airbnb-base/rules/style.js#L120-L145
+        indent: ['error', 4, {
```

Comment:
Adding all of these options in the third parameter is

> 3. I noticed a while ago that the `switch () {}` statements in JS always enforced a wrong indentation... looks like that's because we (accidentally) told ESLint to do so. Fixed it in this PR, see indent rule).

Reply:
To explain:

In Airbnb's configuration this indentation rule is configured properly, so that it follows their (and therefore mostly our's) styleguide: https://github.com/airbnb/javascript/blob/60b96d322277c4c71a21a05caba8eb3320e0e3fa/packages/eslint-config-airbnb-base/rules/style.js#L120-L145

The only difference we wanted is to use **4** spaces, instead of **2**. So we were overriding their rule like this:

```
indent: ['error', 4],
```

However, that completely unset all of the option parameters configured in the third parameter - that big object. So for example `SwitchCase: 1` was not set. And therefore fallback to the default, which is `SwitchCase: 0` (see [here](https://eslint.org/docs/rules/indent#options)) - effectively not linting `switch()` statements the way we want to lint them.

To fix that, and fall in line with Airbnb's style guide (and therefore our's as well) again, I copied over the complete configuration. Rather than leaving them all at whatever ESLint deems to be the default.